sw/device/exts/common/flash_link.ld

```diff
@@ -38,11 +38,6 @@ SECTIONS {
    * signature, but for now it's just the entry point at offset zero.
    */
   .flash_header ORIGIN(eflash) : ALIGN(4) {
```

Comment:
Don't you need a `KEEP(*(.flash_header))` section in here?

Reply:
At the moment we don't have any symbols defined there. If that could change then it would be safer to add the `KEEP`, but I was assuming that wasn't the case.

Reply:
I guess the section might still be GCed, so I'll add the KEEP.